kitsune/log_settings.py

```diff
@@ -84,3 +84,4 @@
     )
 
 dictConfig(config)
+logging.captureWarnings(True)
```

Comment:
What effect does this happen in celery and wsgi and `./manage.py runserver`? Seems like it'd be fine, but I'm curious if you tested it out or not and if the other log settings send the warnings anywhere useful.